### PR TITLE
Sync Chinese translation

### PR DESCRIPTION
**Pull request by LiHaiP**

### CODE REVIEW

**Comment by wenningerk** [comment]
test this please

**Review by gao-yan — CHANGES_REQUESTED** [review]

File: po/zh_CN.po [hunk 1]
```diff
@@ -814,6 +835,13 @@ msgid ""
 "*** Deprecated *** Whether to remove stopped resources from the executor"
 msgstr "***不推荐***是否从pacemaker-execd 守护进程中清除已停止的资源"
 
+#: lib/pengine/common.c:209
+msgid ""
+"Values other than default are poorly tested and potentially dangerous. This "
+"option will be removed in a future release."
+msgstr "默认值以外的值测试结果很差，有潜在的风险。"
+"该选项将在未来的版本中删除。"
```
[the remark this answers is not in the document]
Comment (LiHaiP) [reply po/zh_CN.po:843]: I agree with you and I will update

**Review by gao-yan — APPROVED** [review]
LGTM, thanks.

**Review by kgaillot — COMMENTED** [review]
Thanks for the updates! Would you mind rebasing this to avoid the merge commit? If you're not comfortable with git rebase commands, the simplest way may be to copy the changed files into a clean checkout.

**Comment by kgaillot** [comment]
Hi @LiHaiP ,

This still has merge commits, which we prefer to avoid inside a pull request. Also, it's much simpler in the long run to work from a separate branch instead of making changes in the main branch. Here's one way to do that:

First, save the two files you changed so you don't lose your changes:

    mkdir ~/tmp
    cp lib/pengine/common.c ~/tmp
    cp po/zh_CN.po ~/tmp

Then, make your main branch the same as the upstream main branch:

    git remote add upstream https://github.com/ClusterLabs/pacemaker.git
    git reset --hard upstream/main
    git push origin +HEAD:main

Then, make your changes in a new branch, and push those to your pacemaker repository:

    git checkout -b changes
    mv ~/tmp/common.c lib/pengine/
    mv ~/tmp/zh_CN.po po/
    git add lib/pengine/common.c po/zh_CN.po
    git commit -m "Doc: scheduler: update Chinese translation"
    git push origin changes

Finally, you can close this pull request, and open a new one, selecting the "changes" branch from your repository.

While it's a lot this time, it will simplify any future changes you do. Whenever you need to make more changes, you can do your work in the changes branch. When needed, you can synchronize your main branch with upstream by running `git checkout main && git pull upstream main`, then update your changes branch with `git checkout changes && git rebase main` to avoid merge commits.

**Pull request closed without merging**